src/main/java/org/spongepowered/mod/mixin/entity/MixinEntity.java

```diff
@@ -332,4 +337,23 @@ public void setPersistent(boolean persistent) {
         // TODO
     }
 
+    @Override
+    public <T> Optional<T> getData(Class<T> dataClass) {
```

Comment:
Yep, I've simply added these methods so we don't forget to implement them (because nothing complains that they're missing until you call them at runtime).